src/RequirementsChecker.php

```diff
@@ -294,6 +294,26 @@ public function checkUploadMaxFileSize(?string $min = null, ?string $max = null)
         return ($minCheckResult && $maxCheckResult);
     }
 
+    
+    /**
+     * Checks if the php.ini setting 'max_execution_time' exceeds a reliable and tested execution time 
+     * @param string|null $max
+     * @return bool
+     */
+    public function checkMaxExecutionTime(?string $max = null): bool
```

Comment:
Why string if the time is int?

Reply:
Just following the above function checkUploadMaxFileSize which uses strings for min and max. Yes should be int.